### PR TITLE
fix test

### PR DESCRIPTION
**Pull request by 6zz**
I believe the expected should have items with `rating: 1` filtered out

### CODE REVIEW

File: test/21-exclude-things.test.js [hunk 1]
```diff
@@ -52,6 +47,6 @@ describe('Exclude Things', function() {
 
     const excluded = excludeThings(items, excludes);
 
-    assert.equals(JSON.stringify(excluded), JSON.stringify(output));
+    assert.deepEqual(excluded, output);
```
Comment (jasonly) [inline test/21-exclude-things.test.js:50]: ohh does deepEqual work on an array of objects?!

Comment (6zz) [reply test/21-exclude-things.test.js:50]: I believe so.

Comment (jasonly) [reply test/21-exclude-things.test.js:50]: Ohhh ok!

**Review by jasonly — APPROVED** [review]
LGTM!

**Pull request merged**